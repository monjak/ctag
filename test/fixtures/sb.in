1 Green Sun's Zenith
SB: 1 Corrupted Harvester
