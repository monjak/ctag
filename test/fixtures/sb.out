1x [c]Green Sun's Zenith[/c]
SB: 1x [c]Corrupted Harvester[/c]
